docs-site/site.css

```
@@ -0,0 +1,598 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+html {
+  scroll-behavior: smooth;
+  scroll-padding-top: 84px;
+}
+body {
+  background: var(--cream);
+  color: var(--ink);
+  font-family: "Gabarito", system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Gabarito" (font-family-name-quotes)

(font-family-name-quotes)